pkg/session_recording/player.jsx

```diff
@@ -620,6 +634,9 @@
 
                     /* Skip packets we don't output */
                     if (!pkt.is_io || !pkt.is_output) {
+                        this.setState({cols: pkt.width, rows: pkt.height});
+                        this.state.term.resize(pkt.width, pkt.height);
+                        this._transform();
```

Comment:
This will also execute on user input, which it shouldn't do. I think you need to rework this condition to only execute your part on `!pkt.is_io`.